docker/runtime-builder/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM ubuntu:20.04
+
+# Package versions.
+ARG GO_VERSION=1.16.4
+ARG RUST_NIGHTLY_VERSION=2021-05-07
+ARG OASIS_CORE_VERSION=21.2.7
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -qq && apt-get upgrade -qq && apt-get install -qq \
+    git gcc curl jq pkg-config libssl-dev protobuf-compiler && \
+    apt-get autoclean && apt-get autoremove && rm -rf /var/cache/apt/archives/* && \
+    mkdir -p /src /cargo/.rustup
+
+ENV HOME="/root"
+ENV GOPATH="/go"
+ENV CARGO_HOME="/cargo"
+ENV RUSTUP_HOME="/cargo/.rustup"
+ENV PATH="${CARGO_HOME}/bin:${GOPATH}/bin:/usr/local/go/bin:${PATH}"
+ENV SDK_SRC="/src"
```

Comment:
This seems unused within the build script, so unless the user of the image will mount the sdk folder at `/src`, this will be wrong. Maybe just remove the environment variable here.